apps/meteor/app/otr/client/OTR.ts

```diff
@@ -24,6 +24,16 @@ class OTR implements IOTR {
 		this.instancesByRoomId[rid] = otrRoom;
 		return this.instancesByRoomId[rid];
 	}
+
+	closeAllInstances(): void {
+		// Resets state, but doesnt emit events
+		// Other party should receive event and fire events
+		Object.values(this.instancesByRoomId).forEach((instance) => {
```

Comment:
Note: we're not firing events because the `onLogout` callback is emited after the user has logged out. This means that emitting the event & calling `deleteAllOTRMessages` call won't work.

We considered with @yash-rajpal to create a new `beforeLogout` hook similar to `afterLogoutCleanup` callback we have already, that way, we could emit those events right before the user logs out (at least the deleteMessages)